markdown/bestpractices/operating_hawq_bestpractices.html.md.erb

```diff
@@ -23,21 +23,14 @@ under the License.
 
 This topic provides best practices for operating HAWQ, including recommendations for stopping, starting and monitoring HAWQ.
 
-## <a id="best_practice_config"></a>Best Practices for Configuring HAWQ Parameters
+## <a id="task_qgk_bz3_1v"></a>Best Practices Using the Command Line                                                to Start/Stop HAWQ Cluster Members
```

Comment:
That's weird. The space  wasn't even visible in the file, and I had to delete words on either side and re-enter because it didn't think the space existed. I've seen these phantom spaces before, but this is the first one I had trouble deleting.